README.md

```diff
@@ -16,7 +85,19 @@ $ pip install .
 ```
 in the repository root.
 
-### Documentation
+
+### For contributors
+
+To install this project in editable mode, run the commands below from the root directory of the `trieste` repository.
+```bash
+git clone https://github.com/secondmind-labs/trieste.git
+cd trieste
+pip install -e .
```

Comment:
this is incomplete, starting point at best - to do typing, formatting and testing there is a series of additional installation commands...

Reply:
Strictly speaking they only require tox to run. But yes, the current workflow is a mess and hopefully we'll get a chance to fix (or at least improve) it in not too long.